<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gerer_abonnement</class>
 <widget class="QDialog" name="gerer_abonnement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>621</width>
     <height>511</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Ajouter abonné</string>
    </attribute>
    <widget class="QRadioButton" name="radioButtoneleve">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>230</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>eleve</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>310</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>DATE FIN:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string> NOM:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="prenom">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>150</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>380</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ETABLISSEMENT:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nom">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>110</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="resaisir">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>60</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 170, 255);</string>
     </property>
     <property name="text">
      <string>resaisir</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonabonne">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>200</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>abonne</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>190</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string> AGE:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>230</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string> EMAIL:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="prix">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>350</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string> PRENOM:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cin">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>70</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>270</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>DATE DEBUT:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="adresse">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>230</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>350</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>PRIX:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="etablissement">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>390</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="retour">
     <property name="geometry">
      <rect>
       <x>504</x>
       <y>442</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 170, 255);</string>
     </property>
     <property name="text">
      <string>Retour</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="age">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>190</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string> CIN:(6chiffres)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ajouter">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>60</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 170, 255);</string>
     </property>
     <property name="text">
      <string>ajouter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="modifier">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>100</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 170, 255);</string>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QLabel" name="message">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>440</y>
       <width>351</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QDateEdit" name="date_debut">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>270</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="date_fin">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>310</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Afficher les abonnés</string>
    </attribute>
    <widget class="QPushButton" name="afficherEl">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>20</y>
       <width>191</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>eleve</string>
     </property>
    </widget>
    <widget class="QPushButton" name="afficherAb">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>191</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>abonne</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>256</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="trierAb_age">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>trier age</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="trierAb_nom">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>trier  nom</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>90</y>
       <width>251</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="trirEl_age">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>60</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>trier age</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="trirEl_nom">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>60</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>trier  nom</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rechercher">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>400</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="recherchercin">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>410</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>410</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>CIN</string>
     </property>
    </widget>
    <widget class="QPushButton" name="supprimer">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>400</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="retour_2">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>440</y>
       <width>75</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 170, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>400</y>
       <width>55</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/Downloads/loupe.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>390</y>
       <width>55</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/Downloads/delete (1).png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>450</y>
       <width>55</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/Downloads/return.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>400</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/Downloads/pie-chart.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="commandLinkButton_2">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>400</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>commandLinkButton_2</zorder>
    <zorder>label_7</zorder>
    <zorder>label_3</zorder>
    <zorder>label_2</zorder>
    <zorder>afficherEl</zorder>
    <zorder>afficherAb</zorder>
    <zorder>tableView_2</zorder>
    <zorder>trierAb_age</zorder>
    <zorder>trierAb_nom</zorder>
    <zorder>tableView</zorder>
    <zorder>trirEl_age</zorder>
    <zorder>trirEl_nom</zorder>
    <zorder>rechercher</zorder>
    <zorder>recherchercin</zorder>
    <zorder>label_14</zorder>
    <zorder>supprimer</zorder>
    <zorder>retour_2</zorder>
    <zorder>label_15</zorder>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Statistiques</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
